src/stats/lexd.rs

```diff
@@ -0,0 +1,57 @@
+use std::collections::HashSet;
+
+use tree_sitter::{Language, Parser, TreeCursor};
+
+use rocket_contrib::{json, json::JsonValue};
+
+use slog::Logger;
```

Comment:
```suggestion
use tree_sitter::{Language, Parser, TreeCursor};
use rocket_contrib::{json, json::JsonValue};
use slog::Logger;
```
(current convention in this repo is to group together all the third-party deps)